assets/js/app/editor/Components/Slug.vue

```diff
@@ -61,48 +63,62 @@ export default {
   data: () => {
     return {
       edit: false,
+      locked: false,
       buttonText: 'Locked',
       icon: 'lock',
     };
   },
   mounted() {
     setTimeout(() => {
-      const title = document.querySelector(
-        `input[name='fields[${this.generate}]']`,
-      ).value;
+      console.log('time, time, time');
```

Comment:
Oops, should this be here? @bobdenotter

Reply:
Nope! I ninja-edited that to be gone from master. 🙈